frontend/src/views/web/auth/signup.html

```diff
@@ -30,13 +30,13 @@
         </div>
         <!-- password -->
         <div class="input-field align-left">
-            <input type="{{canShowPassword ? 'text' : 'password'}}" id="password" onpaste="return false;" class="dark-autofill" name="password" ng-model="auth.regUser.password" ng-minlength="8" ng-change="auth.checkStrength(auth.regUser.password)" required>
+            <input type="{{canShowPassword ? 'text' : 'password'}}" id="password" onpaste="return false;" class="dark-autofill" name="password" ng-model="auth.regUser.password" ng-minlength="8" ng-change="auth.checkStrength(auth.regUser.password)">
```

Comment:
We need the required attribute to make sure the user enters a password in the form.

Reply:
You can consider the email input field with the tag required.